types/es-abstract/5/SameValue.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function SameValue(x: unknown, y: unknown): boolean;
```

Comment:
similarly, this should be a predicate that asserts that `x` and `y` have the same type

Reply:
**TypeScript** doesn’t currently expose this functionality, which is why `Object.is` is defined as returning `boolean`.

Reply:
it can't assert that `x is typeof y`?